### PR TITLE
Feature multi binding test

### PR DESCRIPTION
**Pull request by illfang**
added multi binding test for testing bindings with multiple contexts and dynamic context creation / destruction

- one browser, multiple changing contexts
- also added a link to the popuptest

#1315

### CODE REVIEW

**Comment by amaitland** [comment]
Is there a need for this `PR`?

**Comment by illfang** [comment]
If you do not want to integrate #1319

File: CefSharp.Example/Resources/MultiBindingTest.html [hunk 1]
```diff
@@ -0,0 +1,57 @@
+﻿<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">
+<html>
+	<head>
+		<title>Multi Binding Test</title>
```
Comment (amaitland) [inline CefSharp.Example/Resources/MultiBindingTest.html]: Can fix the formatting please :+1:

**Comment by amaitland** [comment]
> If you do not want to integrate #1319

Fair enough :smile: 

Just need to reformat the `Html`, then I'm happy to merge this.

**Comment by illfang** [comment]
I've adjusted the format to use spaces.

**Comment by amaitland** [comment]
Thanks :+1:

**Pull request merged**